src/Behat/Behat/HelperContainer/BuiltInServiceContainer.php

```diff
@@ -115,13 +115,8 @@ private function getAndValidateServiceSchema($id)
 
     /**
      * Gets and validates a class from schema.
-     *
-     * @param string       $id
-     * @param string|array $schema
-     *
-     * @return string
      */
-    private function getAndValidateClass($id, array $schema)
+    private function getAndValidateClass(string $id, array $schema): string
```

Comment:
The annotation for the $schema parameter was wrong. I changed this to use proper php types everywhere